src/styles/blocks/card.scss

```
@@ -0,0 +1,103 @@
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  max-width: 200px;
+  margin: 0 24px 46px;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+    margin-bottom: 40px;
+  }
+
+  &__product-description {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0 16px 18px;
+  }
+
+  &__title {
+    display: inline-block;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    padding-bottom: 5px;
+    color: hsl(234, 80%, 12%);
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    color: rgb(97, 96, 112);
+  }
+
+  &__rating {
+    display: flex;
+    width: 100%;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    padding-bottom: 24px;
+  }
+
+  &__reviews {
+    position: relative;
+    font-size: 10px;
+    font-weight: 400;
+    color: rgb(6, 11, 53);
+    padding-right: 16px;
+  }
+
+  &__price {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    padding-bottom: 16px;
+
+    &-text {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: rgb(97, 96, 112);
+      padding-left: 16px;
+    }
+
+    &-number {
+      font-size: 16px;
+      font-weight: 700;
+      color: rgb(6, 11, 53);
+      padding-right: 16px;
+      line-height: 18px;
+    }
+  }
```

It's not a modifiers or elements of `price`, it's a card block elements.
 More detail in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)

And more information about nesting in this [video](https://mate.academy/learn/css-advanced/sass#/video/925)
```suggestion
  &__price {
    display: flex;
    width: 100%;
    justify-content: space-between;
    padding-bottom: 16px;
  }
  
    &__price-text {
      font-size: 12px;
      font-weight: 400;
      line-height: 18px;
      color: rgb(97, 96, 112);
      padding-left: 16px;
    }

    &__price-number {
      font-size: 16px;
      font-weight: 700;
      color: rgb(6, 11, 53);
      padding-right: 16px;
      line-height: 18px;
    }
```